styles/_base.scss

```diff
@@ -117,48 +117,107 @@ ul, ol {
 
   #contentFrame {
     position: relative;
-    padding-top: $barLg;
     overflow-y: scroll;
     height: calc(100% - #{$bar});
     box-sizing: border-box;
     z-index: 0;
 
-    #pageTabBar {
-      position: fixed;
-      top: $bar;
-      left: 0;
-      z-index: 1;
-      width: 100%;
-      height: $barLg;
+    &:before {
+      // bottom border for the chat close button, it gets covered by the pageTabBar
+      content: '';
+      display: block;
+      width: $chatCloseBtn;
+      height: 0;
       border-bottom-width: 1px;
       border-bottom-style: solid;
+      position: absolute;
+      top: $chatCloseBtnTop - $bar;
+      right: -$chatCloseBtn;
+      width: $chatCloseBtn;
+      height: $chatCloseBtn;
+      @include chatOpeningTransition(right);
+      z-index: 0;
+    }
+
+    .chatOpen &:before {
+      right: 0;
+    }
 
-      .pageTabs {
-        max-width: $pageWidth;
-        margin: 0 auto;
+    #pageContainer {
+      padding-top: $barLg;
+
+      #pageTabBar {
+        position: fixed;
+        top: $bar;
+        left: 0;
+        z-index: 1;
+        width: 100%;
+        height: $barLg;
+        overflow: hidden;
+
+        &:before {
+          // mask for the chat close button
```

Comment:
This is a hole through which the page background can show, the exact same size and shape as the close button.